crowbar_framework/app/controllers/nodes_controller.rb

```diff
@@ -427,7 +427,7 @@ def get_node_and_network(node_name)
     @node = NodeObject.find_node_by_name(node_name) if @node.nil?
     if @node
       # If we're in discovery mode, then we have a temporary DHCP IP address.
-      if not ["discovering", "discovered", "hardware-installing", "hardware-installed"].include? @node.state
+      if not ["discovering", "discovered"].include? @node.state
```

Comment:
Style/Not: Use `!` instead of `not`. (https://github.com/bbatsov/ruby-style-guide#bang-not-not)